python/ct/client/db/cert_desc.py

```diff
@@ -94,7 +96,15 @@ def from_cert(certificate, observations=[]):
 
 
 def to_unicode(str_):
```

Comment:
nit: Odd name for a parameter (suffixed with '_'). Consider changing the parameter name as you've changed the entire function.